packages/react/src/component-tests/IcPageHeader/IcPageHeader.cy.tsx

```diff
@@ -0,0 +1,282 @@
+        reverseOrder
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+        <IcButton slot="actions" variant="primary">
+          Create coffee
+        </IcButton>
+        <IcButton slot="actions" variant="tertiary">
+          Filter coffee
+        </IcButton>
+        <IcTextField
+          slot="input"
+          placeholder="Search for ingredients…"
+          label="Input"
+          hideLabel
+        />
+        <IcNavigationItem slot="tabs" label="Method" href="#" selected />
+        <IcNavigationItem slot="tabs" label="Ingredients" href="#" />
+      </IcPageHeader>
+    );
+    cy.get("ic-navigation-item").should(BE_VISIBLE);
+    cy.viewport(1450, 500);
+    cy.compareSnapshot({
+      name: "withActionInputTab",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+    cy.checkA11yWithWait();
+  });
+
+  it("page header with actions,input and stepper", () => {
+    mount(
+      <IcPageHeader
+        heading="latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam."
+        reverseOrder
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+        <IcButton slot="actions" variant="primary">
+          Create coffee
+        </IcButton>
+        <IcButton slot="actions" variant="tertiary">
+          Filter coffee
+        </IcButton>
+        <IcStepper slot="stepper">
+          <IcStep stepTitle="Warm kettle" stepType="completed" />
+          <IcStep
+            stepTitle="Warm milk"
+            stepSubtitle="Optional"
+            stepType="completed"
+          />
+          <IcStep stepTitle="Pour milk" stepType="current" />
+        </IcStepper>
+        <IcTextField
+          slot="input"
+          placeholder="Search for ingredients…"
+          label="Input"
+          hideLabel
+        />
+      </IcPageHeader>
+    );
+    cy.get("ic-chip").should(BE_VISIBLE);
+    cy.get("ic-button").should(BE_VISIBLE).should(HAVE_LENGTH, "2");
+    cy.get("ic-text-field").should(BE_VISIBLE);
+    cy.viewport(1450, 500);
+    cy.compareSnapshot({
+      name: "withActionInputStep",
+      testThreshold: DEFAULT_TEST_THRESHOLD + 0.05,
+    });
+  });
+
+  it("page header With breadcrumb navigation", () => {
+    mount(
+      <IcPageHeader
+        heading="Latte recipe"
+        subheading="A Latte is a popular Italian coffee, made with espresso, steamed milk and a thin layer of foam."
+      >
+        <IcChip slot="heading-adornment" label="BETA" size="large" />
+        <IcBreadcrumbGroup slot="breadcrumbs">
+          <IcBreadcrumb pageTitle="Drinks" href="#" />
+          <IcBreadcrumb pageTitle="Coffees" href="#" />
+          <IcBreadcrumb current pageTitle="Latte" href="#" />
+        </IcBreadcrumbGroup>
+      </IcPageHeader>
+    );
+    cy.viewport(1450, 500);
```

Comment:
Can the test using this viewport be added to a new describe block and use the viewport within `beforeEach`?